test/index_test.js

```diff
@@ -191,6 +202,14 @@ describe('Client', () => {
 
         balance.should.be.a.Number();
       });
+
+      it('should support named parameters', async () => {
+        const client = new Client(_.defaults({ version: '0.15.0' }, config.bitcoind));
+        const mainWalletBalance = await client.getBalance({ account: '*', minconf: 0 });
+        const testWalletBalance = await client.getBalance({ account: 'test', minconf: 0 });
+
+        mainWalletBalance.should.not.equal(testWalletBalance);
```

Comment:
WDYT of adding an extra assert to test that, for legacy reasons:
`client.getBalance({ account: '*', minconf: 0 })` equals `client.getBalance('*', 0);`